gluster-block.spec.in

```diff
@@ -57,16 +57,19 @@ storage creation and maintenance as simple as possible.
 %if ( 0%{?_with_initd:1} )
 mv %{buildroot}/%{_initddir}/gluster-blockd.initd %{buildroot}/%{_initddir}/gluster-blockd
 %endif
+touch %{buildroot}%{_sharedstatedir}/gluster-block/gb_upgrade.status
 
 %if ( 0%{?_with_systemd:1} )
 %post
+%systemd_post gluster-block-target.service
 %systemd_post gluster-blockd.service
 
 %preun
+%systemd_preun gluster-block-target.service
 %systemd_preun gluster-blockd.service
-rm -rf %{_sharedstatedir}/gluster-block/gb_upgrade.status
```

Comment:
Everytime we uninstall or upgrade, we want to remove this file from the host. How do we do this if we remove this line ?

Reply:
This is what the %ghost will help us to do automatically in %preun section.